NYPLAudiobookToolkit/Core/OverdriveSpineElement.swift

```diff
@@ -0,0 +1,54 @@
+enum OverdriveSpineElementMediaType: String {
+    case audioMP3 = "audio/mp3"
+}
+
+final class OverdriveSpineElement: SpineElement {
+    
+    lazy var downloadTask: DownloadTask = {
+        return OverdriveDownloadTask(spineElement: self)
+    }()
+    
+    lazy var chapter: ChapterLocation = {
+        return ChapterLocation(number: self.chapterNumber,
+                               part: 0,
+                               duration: self.duration,
+                               startOffset: 0,
+                               playheadOffset: 0,
+                               title: self.title,
+                               audiobookID: self.audiobookID)!
+    }()
+    
+    let key: String
+    let chapterNumber: UInt
+    let title: String
+    let url: URL
+    let mediaType: OverdriveSpineElementMediaType
+    let duration: TimeInterval
+    let audiobookID: String
+    
+    public init?(JSON: Any?, index: UInt, audiobookID: String) {
```

Comment:
this probably doesn't need to be `public` since the class is internal?

Can you double-check the rest of the added methods/vars to see if their access can be restricted? I have no idea what the client apps will need, so it's definitely possible this is all already good as-is.